src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -2954,47 +2899,47 @@ void Instruction::execute() {
       break;
     }
     case Opcode::AArch64_LDNPSi: {  // ldnp st1, st2, [xn, #imm]
-      results[0] = memoryData[0].zeroExtend(4, 16);
-      results[1] = memoryData[1].zeroExtend(4, 16);
+      results[0] = memoryData[0].zeroExtend(4, 256);
+      results[1] = memoryData[1].zeroExtend(4, 256);
```

Comment:
Again, unless I am interpreting this wrong - zero extending up to 256 bytes?

Reply:
That's correct. We no longer need to zero extend at the end of the execute function as they were being done previously.